frontend/app/module/election/page/ElectionStatusPage.tsx

```diff
@@ -67,21 +67,28 @@ function getTableHeaderForCategory(category: StatusCategory): ReactNode {
       {t("progress")}
     </Table.Column>
   );
-  // TODO: Needs to be updated when second entry is implemented
-  if (category === "unfinished") {
-    return <CategoryHeader />;
-  } else if (category === "in_progress") {
-    return <CategoryHeader>{[progressColumn]}</CategoryHeader>;
-  } else if (category === "definitive") {
-    return <CategoryHeader>{[finishedAtColumn]}</CategoryHeader>;
-  } else {
-    return <CategoryHeader></CategoryHeader>;
+
+  switch (category) {
+    case "unfinished":
+      return <CategoryHeader />;
+    case "in_progress":
+      return <CategoryHeader>{[progressColumn]}</CategoryHeader>;
+    case "first_entry_finished":
+    case "definitive":
+      return <CategoryHeader>{[finishedAtColumn]}</CategoryHeader>;
+    default:
+      return <CategoryHeader></CategoryHeader>;
```

Comment:
```suggestion
      return <CategoryHeader />;
```
Should have been this already in my code haha